.gitignore

```diff
@@ -1,15 +1,13 @@
 /dist
-gen.go
+/autogen/gen.go
```

Comment:
I think we could now include generated files in git. This will allow to `go get` Traefik. WDYT? But that wan be done in another PR.